src/bucket.js

```diff
@@ -98,6 +99,7 @@ Bucket.prototype.createSignedGetUrl = function(prefix, options) {
   assert(prefix,                'prefix must be given');
   assert(options,               'options must be given');
   assert(options.expires,       'expires must be given');
+  prefix = encodeURIComponent(prefix);
   return new Promise((accept, reject) => {
     this.s3.getSignedUrl('getObject', {
       Key:          prefix,
```

Comment:
Won't aws-sdk do the right thing here?

Reply:
no.  I keep saying that aws-sdk is not that spectacular.